mne/bem.py

```diff
@@ -1158,10 +1159,11 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
             rr, tris, volume_info = read_surface(surf_ws_out,
                                                  read_metadata=True)
-            volume_info.update(new_info)  # replace volume info, 'head' stays
+            # replace volume info, 'head' stays
+            volume_info.update(new_info)
+            write_surface(surf_ws_out, rr, tris, volume_info=volume_info,
+                          overwrite=True)
```

Comment:
`overwrite=overwrite` ?

Reply:
@agramfort Since it overwrites original surfaces computed by FreeSurfer, `overwrite` should be True. Whether to overwrite the existing surfaces will be checked in [previous lines](https://github.com/mne-tools/mne-python/blob/412ce469f55528b9657847fd3aea78aa2b1709c2/mne/bem.py#L1113)